src/holochainConfig.js

```diff
@@ -1,5 +1,5 @@
 export const DEVELOPMENT_INSTANCE_NAME = 'acorn'
 export const ZOME_NAME = 'holo_acorn'
 export const DEFAULT_HOLOCHAIN_PORT = '8888'
-export const DEFAULT_HOLOCHAIN_HOST = '127.0.0.1'
+export const DEFAULT_HOLOCHAIN_HOST = '192.168.1.63'
 
```

Comment:
If you change this, don't git commit it, because it breaks other peoples development setups